test/cpp/end2end/generic_end2end_test.cc

```diff
@@ -82,10 +82,12 @@ class GenericEnd2endTest : public ::testing::Test {
     bool ignored_ok;
     cli_cq_.Shutdown();
     srv_cq_->Shutdown();
-    while (cli_cq_.Next(&ignored_tag, &ignored_ok))
+    while (cli_cq_.Next(&ignored_tag, &ignored_ok)) {
       ;
-    while (srv_cq_->Next(&ignored_tag, &ignored_ok))
+    }
+    while (srv_cq_->Next(&ignored_tag, &ignored_ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done